dev-docs/source/WritingPerformanceTests.rst

```diff
@@ -174,7 +174,7 @@ easily monitor timings. The job runs a set of `performance tests on the
 main branch of
 Mantid <http://builds.mantidproject.org/job/main_performancetests2/>`__.
```

Comment:
This link does not appear to work

Reply:
I wanted to avoid touching that section because it's completely out of date and we need to consider what to do about performance tests. I've updated it now so it describes the current situation.